lib/flushbar.dart

```diff
@@ -347,7 +347,7 @@ class _FlushbarState<K extends Object?> extends State<Flushbar<K>>
   }
 
   void _configureLeftBarFuture() {
-    SchedulerBinding.instance.addPostFrameCallback(
+    SchedulerBinding.instance!.addPostFrameCallback(
```

Comment:
Should be a ? instead of a !

Reply:
Yeah, it should be?

Reply:
If you do ! you are saying "i know for a 100% that SchedulerBinding.instance is not null. I don't think we know that. If you use ? you are saying "only call the addPostFrameCallback method if SchedulerBinding.instance is not null". It adds null-safety. So therefore I am sure that ? is the correct answer here and ! is wrong.